EventFilter/L1TRawToDigi/plugins/implementations_stage2/EGammaUnpacker.h

```diff
@@ -7,7 +7,16 @@ namespace l1t {
    namespace stage2 {
       class EGammaUnpacker : public Unpacker {
          public:
+            EGammaUnpacker();
+            ~EGammaUnpacker() override {};
+
             bool unpack(const Block& block, UnpackerCollections *coll) override;
+
+            inline void setEGammaCopy(const unsigned int copy) { EGammaCopy_ = copy; };
+
+         private:
+            EGammaBxCollection* res_;
```

Comment:
res_ is not used since a variable with the same name is also declared in the .cc. This one can be removed.